app/Http/Controllers/Api/RequestApiController.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+
+use App\Http\Controllers\Controller;
+
+class RequestApiController extends Controller
+{
+    /**
+     * Create a new controller instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+    }
+
+    // Function that uses the given url to return the resolved headers
+    public function check($url)
+    {
+        //Retrieve the headers and store in an array
+        $header = $this->getAllHeaders($url);
+        return view
+        ('master', [
+            'url' => $url,
+            'header' => $header
+        ]);
+    }
+
+    // Function that retrieves the headers and returns them in form of an array
+    public function getAllHeaders($url)
```

Comment:
`getAllHeaders` is not called as a public method. You can change `public` to `private`.